datetimerange/__init__.py

```diff
@@ -635,6 +635,61 @@ def intersection(self, x):
             end_time_format=self.end_time_format,
         )
 
+    def subtract(self, x):
+        """
+        Remove a time range from this one and return the result.
+        The result will be [self.copy()] if the second range does not overlap the first
+        The result will be [] if the second range wholly encompasses the first range
+        The result will be [new_range] if the second range overlaps one end of the range
+        The result will be [new_range1, new_range2] if the second range is an internal sub range of the first
```

Comment:
The above lines are a little bit hard to read in the HTML after the docs build.
How about modifying like the followings:

```
        Remove a time range from this one and return the result.

        - The result will be ``[self.copy()]`` if the second range does not overlap the first
        - The result will be ``[]`` if the second range wholly encompasses the first range
        - The result will be ``[new_range]`` if the second range overlaps one end of the range
        - The result will be ``[new_range1, new_range2]`` if the second range is an internal sub range of the first
```